docs/reference/rest-api/logs.asciidoc

```diff
@@ -0,0 +1,278 @@
+<<dynamic-mapping-runtime-fields, maps dynamic fields as runtime fields>>.
+
+For custom fields that are frequently used in searches or aggregations, you might want to leverage the speed benefits of indexing the field.
+This comes at the expense of a larger index size more processing at ingest time.
+To index a custom field, <<create-index-template, create an index template>> for your dataset `logs-<dataset>-*` based on the built-in index template for `logs-*-*` and add your custom field to the mappings.
+To immediately apply the setting, <<manually-roll-over-a-data-stream, roll over the data stream>>
+This affects any new data added to the stream after the rollover.
+However, it does not affect the data stream’s existing backing indices or existing data.
+
+All fields, aside from the `@timestamp` field, are configured to <<ignore-malformed, ignore malformed>> values.
+This means that if a log event contains a field whose type is incompatible with the type of the field that exists in the mapping,
+{es} will ignore the field instead of rejecting the whole document.
+For example, when a string is provided for a field that is mapped to integer.
+Note that this currently doesn't apply for object/scalar mismatches, such as `"foo": "bar"` vs `"foo.bar": "baz"`.
+
+[discrete]
+[[logs-api-path-params]]
+=== {api-path-parms-title}
+
+{ecs-ref}/ecs-data_stream.html#field-data-stream-dataset[`data_stream.dataset`]::
+  (Optional, string)
+  Defaults to `generic`.
+  Describes the ingested data and its structure.
+  It is highly recommended to provide a value for this so that you can add structure to your logs after the fact.
+  Example: `nginx.access`.
+
+{ecs-ref}/ecs-data_stream.html#field-data-stream-namespace[`data_stream.namespace`]::
+  (Optional, string)
+  Defaults to `default`.
+  A user-configurable arbitrary grouping, such as an environment (dev, prod, or qa), a team, or a strategic business unit.
+
+[discrete]
+[[logs-api-query-params]]
+=== {api-query-parms-title}
+
+Any provided query parameter will be added to each log line.
+For example, `/_logs?service.name=myapp` will add `"service.name": "myapp"` to all logs.
+
+[discrete]
+[[logs-api-request-body]]
+=== {api-request-body-title}
+The request body contains a newline-delimited list of log events to ingest.
+The individual events don't have any required fields and can contain arbitrary JSON content.
+There is no required structure for the log events, and you can add any top-level fields.
+However, it is recommended to follow the {ecs-ref}[Elastic Common Schema (ECS)] to structure your logs.
+
+TIP: Use the {ecs-logging-ref}/intro.html[ECS logging libraries] to create ECS-compliant JSON logs.
+
+Only the following fields, which are commonly used for searching, filtering, and correlating logs, are indexed by default:
+
+* {ecs-ref}/ecs-base.html#field-timestamp[`@timestamp`] +
```

Comment:
The list is pretty long. Maybe we can get it even shorter?

Reply:
Shorter, because it takes up a lot of space in the docs or because you think some of the fields should not be indexed by default? If so, which ones and why?